rgo/component.py

```diff
@@ -30,14 +30,22 @@
 SRPM_RE = re.compile(r".+\.(?:no)?src\.rpm")
 
 class Component(object):
-    def __init__(self, name, git=None, distgit=None, requires=[]):
+    def __init__(self, name, version_from=None, git=None, distgit=None, requires=[]):
         """
         :param str name: Name of component
         :param rgo.git.Git: Git repository
         :param rgo.git.DistGit: Dist-Git repository
         """
         assert git or distgit
         self.name = name
+        self.version_from = version_from
+
+        if self.version_from is None:
+            if self.distgit:
```

Comment:
At this stage `self.distgit` isn't assigned:
```
...
  File "/home/lu/dev/rpm-gitoverlay/rgo/component.py", line 44, in __init__
    if self.distgit:
AttributeError: 'Component' object has no attribute 'distgit'
```

Reply:
Fixed